File: apps/problems/Software-Dev/boilerplate-full/function.cpp
#include <iostream>
    #include <fstream>
    #include <vector>
    #include <string>
    #include <sstream>
    #include <climits>
    
    ##USER_CODE_HERE##
    
    int main() {
      std::ifstream file("/dev/problems/Software-Dev/tests/inputs/##INPUT_FILE_INDEX##.txt");
      std::vector<std::string> lines;
      std::string line;
      while (std::getline(file, line)) lines.push_back(line);
    
      file.close();
      int F;
  std::istringstream(lines[0]) >> F;
  int B;
  std::istringstream(lines[1]) >> B;
  int D;
  std::istringstream(lines[2]) >> D;
      std::int result = solve(F, B, D);
      std::cout << result << std::endl;
      return 0;
    }
    

File: apps/problems/Software-Dev/boilerplate-full/function.rs
use std::fs::read_to_string;
    use std::io::{self};
    use std::str::Lines;
    
    ##USER_CODE_HERE##
    
    fn main() -> io::Result<()> {
      let input = read_to_string("/dev/problems/Software-Dev/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
      let mut lines = input.lines();
      let F: i32 = lines.next().unwrap().parse().unwrap();
  let B: i32 = lines.next().unwrap().parse().unwrap();
  let D: i32 = lines.next().unwrap().parse().unwrap();
      let result = solve(F, B, D);
      println!("{}", result);
      Ok(())
    }
    